src/main/java/org/mariadb/jdbc/internal/query/MariaDbQuery.java

```diff
@@ -117,8 +118,9 @@ public int writeLastRewritePartLength() {
      */
     public void writeToRewritablePart(final OutputStream os, int rewriteOffset) throws IOException {
```

Comment:
I agree with you, If i can modify the API, I will do it for the next commit.

Thanks

On Fri, Mar 4, 2016 at 3:35 AM, Mike Jensen <notifications@github.com>
wrote:

> In src/main/java/org/mariadb/jdbc/internal/query/MariaDbQuery.java
> <https://github.com/MariaDB/mariadb-connector-j/pull/72#discussion_r54983425>
> :
>
> > @@ -117,8 +118,9 @@ public int writeLastRewritePartLength() {
> >       */
> >      public void writeToRewritablePart(final OutputStream os, int rewriteOffset) throws IOException {
>
> Why did you not just update the API to accept PacketOutputStream?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MariaDB/mariadb-connector-j/pull/72/files#r54983425>.
>